pokeapp/src/components/PokemonCard.js

```diff
@@ -56,6 +55,27 @@ const PokemonCard = ({
     }
   };
 
+  const removeFromPokeList = (pokemon, e) => {
+    //Finds the index of pokemon in the array that matches the id of the pokemon being added to the array
+    const pokeIndex = myPokeList.findIndex(
+      (element) => element.id === pokemon.id
+    );
+    let tempPokeList = [...myPokeList];
+
+    //If the pokemon is already in the array
+    if (pokeIndex !== -1) {
+      if (tempPokeList[pokeIndex].quantity > 1) {
+        tempPokeList[pokeIndex].quantity--;
+      } else if (tempPokeList[pokeIndex].quantity === 1) {
+        tempPokeList[pokeIndex].quantity--;
+        tempPokeList = tempPokeList.filter(
+          (pokemon) => pokemon.id !== parseInt(e.target.value)
+        );
+        setMyPokeList(tempPokeList);
+      }
+    }
+  };
```

Comment:
It looks like it works!!! (me thinks................)

Reply:
It does!